packages/data-store/src/utils/presentationDefinition/MappingUtils.ts

```diff
@@ -0,0 +1,59 @@
+import { PresentationDefinitionItemEntity } from '../../entities/presentationDefinition/PresentationDefinitionItemEntity'
+import { IPresentationDefinition } from '@sphereon/pex'
+import { PartialPresentationDefinitionItem, PresentationDefinitionItem } from '../../types'
+import md5 from 'md5'
+
+export const presentationDefinitionItemFrom = (entity: PresentationDefinitionItemEntity) => {
+  const item: PresentationDefinitionItem = {
+    id: entity.id,
+    tenantId: entity.tenantId,
+    definitionId: entity.definitionId,
+    version: entity.version,
+    name: entity.name,
+    purpose: entity.purpose,
+    definitionPayload: JSON.parse(entity.definitionPayload) as IPresentationDefinition,
+    createdAt: entity.createdAt,
+    lastUpdatedAt: entity.lastUpdatedAt,
+  }
+  return item
+}
+
+export const presentationDefinitionEntityItemFrom = (item: PartialPresentationDefinitionItem) => {
+  const entity = new PresentationDefinitionItemEntity()
+  if (item.id) {
+    entity.id = item.id
+  }
+
+  entity.tenantId = item.tenantId
+  entity.definitionId = item.definitionId!
+  entity.version = item.version!
+  entity.name = item.name
+  entity.purpose = item.purpose
+  entity.definitionPayload = JSON.stringify(item.definitionPayload!)
+  entity.createdAt = item.createdAt!
+  entity.lastUpdatedAt = item.lastUpdatedAt!
+  return entity
+}
+
+function hashPayload(payload: IPresentationDefinition): string {
+  return md5(JSON.stringify(payload))
```

Comment:
Please use the same algo we are already using for VCs and VPs in Veramo (search for the createEntryHash function). Actually that function should already be capable of creating the hash, but might introduce some Veramo deps. Point is I rather not see md5 being used and rather use the same hash algo as used elsewhere

Reply:
@nklomp 
I looked at computeEntryHash, but it seems to want W3CVerifiableCredential | W3CVerifiablePresentation | IMessage
computeEntryHash is using blake2bHex(hashable) under the hood.
So I needed to introduce either dep blakejs or md5. The latter was <200 lines of code and seemed adequate. But I can switch to blake2bHex

Reply:
Md5 is broken for many many years now. We already include the other dep, so do not introduce new ones

Reply:
Fixed

Reply:
@nklomp please confirm if ok now, then I can merge